### PR TITLE
Project 2: Add reducers and actions.

### PR DESCRIPTION
**Pull request by binhussen**
## Project 2: Add reducers and actions.
In This PR, I have implemented
- Add Redux Toolkit.
- Structure Redux files/folder using [ducks pattern](https://github.com/erikras/ducks-modular-redux).
- Create Actions and Reducers for books and categories.
- Configure the Redux Store.

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+/** @format */
+export const CREATE_BOOK = 'bookstore/books/CREATE';
+export const REMOVE_BOOK = 'bookstore/books/REMOVE';
```
Comment (FrederickMih) [inline src/redux/books/books.js:3]: - [Optional] I think the pattern should end with the action type that's:

```
export const CREATE_BOOK = 'bookstore/books/CREATE_BOOK';
export const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
```

**Review by FrederickMih — APPROVED** [review]
Hi @binhussen , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights :green_circle: 

- Redux store has been properly configured :rocket: 
- README.md file is descriptive and professional :+1: 
- Pull Request is professional :+1: 
- Great commit history :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/books.js (hunk 1 above)
Comment (binhussen) [reply src/redux/books/books.js:3]: Thanks for your feedback, I will modify it.

**Comment by binhussen** [comment]
Thank you @FrederickMih for your feedback and approval.

**Pull request merged**